src/main/java/com/cms/claimmanagement/insuranceCompany/repository/claimResponseData.java

```diff
@@ -0,0 +1,4 @@
+package com.cms.claimmanagement.insuranceCompany.repository;
+
+public record claimResponseData() {
```

Comment:
Java convention suggests that classes names should start with upper case. :)

Reply:
It is, but maybe not right now. Surveyor is assingned a Claim and SurveyReport has claimId as its key. But I think it would make more sense to push it with the Insurance Company module.

Reply:
It's fine. I would just update the name to ClaimResponseData to follow the convention.